p2p/src/authenticated/discovery/actors/tracker/set.rs

```diff
@@ -84,9 +84,9 @@ mod tests {
 
     fn expected_sorted_peers() -> Vec<ed25519::PublicKey> {
         vec![
+            ed25519::PrivateKey::from_seed(1).public_key(), // ff87a0b0a3c7c0ce827e9cada5ff79e75a44a0633bfcb5b50f99307ddb26b337
```

Comment:
nit: I suspect you had to change the order because the commented addresses are no longer the same. Can you update (just print out to terminal and update).

Reply:
done